GPU/GPUTracking/Base/GPUReconstructionConvert.cxx

```diff
@@ -872,6 +874,12 @@ void zsEncoderDenseLinkBased::decodePage(std::vector<o2::tpc::Digit>& outputBuff
 
       const unsigned char* pageNext = ((const unsigned char*)decPage) + TPCZSHDR::TPC_ZS_PAGE_SIZE;
       const o2::header::RAWDataHeader* rdhNext = (const o2::header::RAWDataHeader*)pageNext;
+
+      if (o2::raw::RDHUtils::getPacketCounter(*rdh) + 1 != o2::raw::RDHUtils::getPacketCounter(*rdhNext)) {
```

Comment:
@fweig : this check must also be added to the GPU decoding. It might be that the CRU drops individual packets, and then the next page in memory is not the expected one which contains the rest of the payload. In this case we simply have to stop decoding adc samples from the current page.
Note that in the GPU version you'll probably have to fill invalid ADC values until you have written `nADCsamples` entries.